core/benchmarks/ZStandardBenchmark-results.txt

```diff
@@ -2,26 +2,26 @@
 Benchmark ZStandardCompressionCodec
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 17.0.9+9-LTS on Linux 5.15.0-1051-azure
-AMD EPYC 7763 64-Core Processor
+OpenJDK 64-Bit Server VM 17.0.9+0 on Mac OS X 14.1.2
+Apple M2 Max
```

Comment:
You can regenerate the result in your CI.
- https://github.com/apache/spark/actions/workflows/benchmark.yml

Reply:
Ya, it's pending on the resouces